Kernel/Output/PDF/Ticket.pm

```diff
@@ -184,6 +186,17 @@ sub GeneratePDF {
         );
     }
 
+    if ( $Interface{Customer} ) {
+        %PrintingCustomerData = $CustomerUserObject->CustomerUserDataGet(
+            User => $Param{UserID},
+        );
+    }
+
+    if ( !defined %PrintingCustomerData )
```

Comment:
Checking the definedness of hashes and arrays is not really supported. The documentation says:
            Use of "defined" on aggregates (hashes and arrays) is no longer
            supported. It used to report whether memory for that aggregate
            had ever been allocated. You should instead use a simple test
            for size:

                if (@an_array) { print "has array elements\n" }
                if (%a_hash)   { print "has hash members\n"   }